### DISCUSSION TITLE
Author Metadata in quarto article

### DISCUSSION THREAD

**Discussion by nithinmkp** [quarto-dev/quarto-cli#10838] [Q&A]
### Description

I want to achieve this format for author metadata

Title 

Author 1 (Corresponding) [any author can be corresponding]
Department
Institute

Author 2
Department
Institute

Author 3
Department 
Institute

If authors are from same department and institute, need to mention only once

Tried with latex partial, but could not achieve this

here is my MWE

title.tex 
````{=tex}
$if(title)$
\title{$title$$if(thanks)$\thanks{$thanks$}$endif$}
$endif$

$if(author)$
$for(author)$
\author{$author.name$}
$if(by-author.affiliations)$ % New line for the author's affiliation
    \affil{$by-author.affiliations.name$} % Displays the affiliation
$endif$
\vspace{1.5em} % Adds space between authors
$endfor$
$endif$

\date{$date$}
````

quarto YAML

````{=qmd}
format: 
  pdf:
    documentclass: scrartcl
    template-partials: 
      - title.tex
    include-before-body: 
      - text: |
         \pagenumbering{gobble}
title: "The Excess Sensitivity of Consumption to Sentiments: Does the PIH hold for India?"
author:
  - name: Nithin M
    affil-id: 1
    attributes:
      corresponding: true
      email: write2nithinm@iitkgp.ac.in
    role:
      - conceptualisation
  - name: Siddhartha Chattopadhyay
    affil-id: 1
  - name: Sohini Sahu
    affil-id: 2

affiliations: 
  - id: 1
    name: Department of Humanities and Social Sciences, IIT Kharagpur
  - id: 2
    name: Department of Economic Sciences, IIT Kanpur
````

**Comment by arnaudgallou** [comment]
I'd recommend to use a journal template and modify/add partials from there if that's really needed. You can download templates developed by the Quarto team here: https://github.com/quarto-journals/.

I'd also recommend to read Quarto's [*Authors & Affiliations* page](https://quarto.org/docs/journals/authors.html) for the recommended way to organize author data. If you're an R user, I created a package to handle author data in Quarto: https://arnaudgallou.github.io/plume/. That will ensure that you use a proper formatting and don't have misplaced fields (such as `email` under `attributes`).
